samples/notification-hubs-test-app/app/src/main/java/com/microsoft/notification_hubs_test_app/MainActivity.java

```diff
@@ -18,16 +20,47 @@
     public static Boolean isVisible = false;
     private static final String TAG = "MainActivity";
     private static final int PLAY_SERVICES_RESOLUTION_REQUEST = 9000;
+    private SharedPreferences.OnSharedPreferenceChangeListener onSharedPreferenceChangeListener = null;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
+        onSharedPreferenceChangeListener = new SharedPreferences.OnSharedPreferenceChangeListener() {
+            @Override
+            public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, String key) {
+                if (key == "registrationID") {
+                    String value = sharedPreferences.getString(key, "");
+                    updateRegistrationID(value);
+                }
+                else if (key == "FCMtoken") {
```

Comment:
Nit, move else if to previous line as per Google Java style guide: https://google.github.io/styleguide/javaguide.html

Reply:
:+1: